Miguel Tuñón 8-929-2198/imprin.c

```diff
@@ -0,0 +1,18 @@
+#include<stdio.h>
+void main()
+{
+	int n,f,i=0,c;
+	
+	printf("Ingrese un valor inicial\n");
+	scanf("%d",&n);
+	printf("Ingrese un valor final\n");
+	scanf("%d",&f);
+	printf("Ingrese el contador\n");
+	scanf("%d",&i);
+	
+	for(c=n;c<=f;c+i)
+	{
+		c=c+i;

```

Comment:
entiendo porque lo colocaste aqui. Pero hubiera estado mejor que lo colocaras en la parte de arriba junto a los demas parametros del for

Reply:
hice una prueba donde introduje el valor 0,10 y 3 y llego hasta 11.  Eso pasa porque no colocaste el parametro al final, y entras al ciclo a pesar de haber superar el numero final